ci/jenkins/jobs/projects-lab.yaml

```diff
@@ -1339,3 +1339,189 @@
               default-excludes: true
               fingerprint: false
               only-if-success: false
+      - '{name}-{test_name}-for-pull-request':
+          test_name: kind-ipv6-only-networkpolicy
+          node: 'antrea-kind-testbed'
+          description: 'This is the {test_name} test for {name}.'
+          branches:
+          - ${{sha1}}    
+          builders:
+            - builder-kind-ipv6-ds-conformance:
+               conformance_type: 'networkpolicy'
+               ip_version: 'ipv6'
+          trigger_phrase: ^(?!Thanks for your PR).*/test-kind-ipv6-(networkpolicy|all).*
+          white_list_target_branches: []
+          allow_whitelist_orgs_as_admins: true
+          admin_list: '{antrea_admin_list}'
+          org_list: '{antrea_org_list}'
+          white_list: '{antrea_white_list}'
+          only_trigger_phrase: true
+          trigger_permit_all: true
+          status_context: jenkins-kind-ipv6-networkpolicy
+          status_url: --none--
+          success_status: Build finished.
+          failure_status: Failed. Add comment /test-kind-ipv6-networkpolicy to re-trigger.
+          error_status: Failed. Add comment /test-kind-ipv6-networkpolicy to re-trigger.
+          triggered_status: null
+          started_status: null
+          wrappers:
+          - timeout:
+              fail: true
+              timeout: 300
```

Comment:
Here timeout is 300, but the cleanup command in builder-kind-ipv6-ds-conformance is using 135 `./ci/kind/kind-setup.sh destroy --all --until 135`.

Maybe it should use maximum of all job timeouts (which is 300) in the cleanup command?

If we don't allow triggering multiple kind jobs in a single Jenkins node (the node slot=1), maybe no need to consider `--until ` when it does cleanup?  Just delete all existing Kind clusters, there must be no other Job but the current Job run using this Jenkins node.

Reply:
Yes , networkpolicy tests are taking `4hr 23 min` to complete its executions that's why I set timeout=300 ,yes If we don't allow triggering multiple kind jobs in a single Jenkins node (i.e the node slot=1) then we don't need `--until` flag.